paasta_tools/paasta_remote_run.py

```diff
@@ -171,6 +172,8 @@ def paasta_to_task_config_kwargs(
         'uris': [uris],
         'docker_parameters': docker_parameters,
     }
+    if gpus > 0:
+        kwargs['gpus'] = float(gpus)
```

Comment:
doing this because task processing and mesos expects floats, even though gpus cannot be partially used.